test/Interop/Cxx/static/global-var-silgen.swift

```diff
@@ -0,0 +1,13 @@
+// Test that global variables are handled properly by the ClangImporter.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -emit-sil -o %t/global-var.sil -Xfrontend -enable-cxx-interop
+// RUN: %FileCheck < %t/global-var.sil %s
+
+// CHECK: %0 = global_addr @counter : $*Int32
```

Comment:
Do you want to also test the code generation pattern for the read and write into the variable?